css/design_system/components/footer.scss

```
@@ -0,0 +1,110 @@
+ /**
+ * @title Simple Footer
+ * @category components
+ * @component_type footer
+ * @value s-footer
+ * @description Simple footer with paragraph text only. The <code>.s-footer-content</code> class should be used on the container that wraps content.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ * </footer>
+ */
+
+.s-footer {
+  @extend .s-p-4;
+  @extend .s-bg-gray-800;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.s-footer-content {
+  * {
+    @extend .s-my-0;
+    @extend .s-text-gray-100;
+    @extend .s-text-start;
+  }
+}
+
+ /**
+ * @title Footer with Links
+ * @category components
+ * @component_type footer
+ * @description Footer with paragraph text and one set of links. Use of the <code>.s-footer-right</code> and <code>.s-footer-link</code> classes ensure proper spacing between links.
+ *
+ * @example
+ * <footer class="s-footer">
+ *  <div class="s-footer-content">
+ *    <p>Turing School of Software and Design, a Colorado Non-Profit Organization.</p>
+ *  </div>
+ *  <div class="s-footer-right">
+ *    <a class="s-footer-link" href="/privacy">Privacy Policy</a>
+ *    <a class="s-footer-link" href="https://turing.edu/programs/" target="blank">Programs</a>
+ *    <a class="s-footer-link" href="https://turing.edu/try-coding/">Try Coding</a>
```

Totally optional, but since this code is just for explanation we don't need to use real links. Less text might keep the focus on just the styles; fewer distractions.
```suggestion
 *    <a class="s-footer-link">Privacy Policy</a>
 *    <a class="s-footer-link">Programs</a>
 *    <a class="s-footer-link">Try Coding</a>
```